### DISCUSSION TITLE
Recently Dumped V.Smile Game Not On List

### DISCUSSION THREAD

**Discussion by lorenzsandi** [mamedev/discussions#32] [Q&A]
Someone recently released a dump of The Wiggles: It's Wiggle Time for the V.Smile and is the US release. The current games list (vsmile_cart.xml) says the game has not been dumped and was wondering if the dumped status of the US version can be updated to dumped. I tested the game and it seems to boot with inputs working. Also I have a few Chinese versions of V.Smile games not yet dumped at all (these being Alphabet Park Adventure, Lion King: Simba's Big Adventure, Toy Story 2: Operation Rescue Woody, and Mickey's Magical Adventure). This is because I don't currently have dumping hardware for VTech consoles.
[Dumped game on Archive.org](https://archive.org/details/wiggles-the-its-wiggle-time-usa)

**Comment by rb6502** [comment]
The best way to resolve this is to open a pull request that adds those games to the software list.  Working on the software lists is a great way to contribute to MAME without being technical.
